kip-0040.md

```diff
@@ -0,0 +1,140 @@
+---
+KIP: "0040"
+Title: kadena_getNetworks_v1 API
+Author: Danillo Felixdaal, Bart Huijgen and Albert Groothedde
+Status: Draft
+Type: Standard
+Category: Interface
+Created: 2025-02-17
+---
+
+# kadena_getNetworks_v1 API
+
+## Abstract
+
+This KIP standardizes the `kadena_getNetworks_v1` method to provide a list of networks known by the wallet. This allows dApps to present the user with a list of available networks and allow them to select one, as well as the ability to submit transactions on these wallet defined networks.
+
+The dapp can use the URL to make /local calls from the network.
+
+The wallet returns only the host. The dapp should append networkId and chainId
+to the URL.
+
+Example:
+
+- Wallet returns: https://api.testnet.chainweb.com
+- Dapp can send requests to:
+  https://api.testnet.chainweb.com/chainweb/0.0/testnet04/chain/19/pact/api/v1/local
+
+> [!NOTE]  
+> For more information about Chainweb API see
+> [Chainweb API Documentation](api.chainweb.com)
```

Comment:
Same comment as KIP 39. This seems out of place in this section. Best to add this after the Response type definition in the `Specification` section.